packages/labs/react/src/test/create-component_test.tsx

```diff
@@ -113,14 +120,15 @@ const BasicElementComponent = createComponent({
 const renderReactComponent = async (
   props?: React.ComponentProps<typeof BasicElementComponent>
 ) => {
-  window.ReactDOM.render(
-    <>
-      <div {...(props as React.HTMLAttributes<HTMLDivElement>)}/>,
-      <x-foo {...props}/>
-      <BasicElementComponent {...props}/>,
-    </>,
-    container
-  );
+  act(() => {
```

Comment:
No action - but this was interesting to see because sometimes folk want a similar thing for Lit. We provide an `updateComplete` promise to await on, but usage of that throughout the tests ends up littering the tests. I wonder if we could provide a similar helper that essentially does a deep, `updateComplete` waiting?

Reply:
Good idea. It could be a nice util to include to `@lit-labs/testing`.